docs/source/conf.py

```diff
@@ -27,7 +27,6 @@
 # ones.
 extensions = [
     "sphinx.ext.todo",
-    "sphinx.ext.intersphinx",
```

Comment:
Why is it being deleted? It allows to cross reference into `dpnp` project, or C-API of `dpctl` or syclinterface.

Reply:
It causes a flood of warnings during sphinx compilation. The issue is `autoapi` uses the same template for all files and then intersphinx finds the same ref in multiple files.

Reply:
added back